English_transcripts/school/Epic Quest/sharedAssets/P_goat_gate.txt

```diff
@@ -0,0 +1 @@
+Feed the goat and tell him to unloack the gate.
```

Comment:
Feed the goat and tell him to unlock the gate.

Reply:
Updated the transcript.